src/app/features/accessibility/accessibility.component.html

```diff
@@ -0,0 +1,86 @@
+        <a href="https://abilitynet.org.uk/" target="_blank">AbilityNet </a> has advice on making your device easier to use if you have a disability. </p>
+    <strong>How accessible is this web application?</strong>
+
+    <div>We know some parts of this website are not fully accessible: <br> &nbsp;&nbsp;&nbsp;
+        • TBC</div> <br>
+
+    <strong>Feedback and contact information</strong>
+    <div>If you need information on this web application in a different format; like accessible PDF, large print, easy
+        read, audio recording or braille:
+    </div>
+    <ul>
+        <li>email customerservices@ukho.gov.uk</li>
+        <li>call +44 (0)1823 484444</li>
+    </ul>
+    <div>We’ll consider your request and reply to you within 5 working days.</div> <br>
+    <strong>Reporting accessibility problems with this web application</strong>
+    <div>
+        We’re always looking to improve the accessibility of this web application. If you find any problems not listed
+        on this page or think we’re not meeting accessibility requirements, contact: customerservices@ukho.gov.uk to let
+        us know your concerns.
+    </div><br>
+    <strong>Enforcement procedure</strong>
+    <div>
+        The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites
+        and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not
+        happy with how we respond to your complaint,
+        <a href="https://www.equalityadvisoryservice.com/" target="_blank">contact the Equality Advisory and Support Service (EASS)</a> .
+    </div><br>
+    <strong>Technical information about this web application’s accessibility</strong>
+    <div>
+        The UK Hydrographic Office is committed to making this website accessible, in accordance with the Public Sector
+        Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
+    </div><br>
+
+    <strong>Compliance status</strong>
+    <div>Select one after testing</div>
+    <div>
+        This website is fully compliant with the Web Content Accessibility Guidelines version 2.1 AA standard.
+        <br>
+        This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard, due
+        to [insert one of the following: ‘the non-compliances’, ‘the exemptions’ or ‘the non-compliances and
+        exemptions’] listed below.
+        <br>
+        This website is not compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The [insert
+        one of the following: ‘non-compliances’, ‘exemptions’ or ‘non-compliances and exemptions’] are listed below.
+    </div> <br>
+    <strong>Non-accessible content</strong>
+
+    <div>The content listed below is non-accessible for the following reasons.
+        Non-compliance with the accessibility regulations
+        <br> &nbsp;&nbsp;&nbsp;
+        • TBC as per testing
```

Comment:
Really minor point but should this be a list item (<ul><li>.......) rather than a carriage return and three non breaking spaces?

Reply:
fixed by addling list item